### PR TITLE
Highlights the active card in HorizontalContentFeeds

### LINKED ISSUE #1114: Implement disabled/focused state for content children/sibling
### User Story

As a user, when I'm visiting an item with siblings, I should see the current item displayed differently (and it shouldn't be tappable).

### Description

The current item in a list of sibling content items (look at a devotional child item for a good example) should be displayed differently than its peers. Tapping on that item should also not result in a navigation action. 

>All it is = entire component >> 50% opacity and remove the drop shadow/
- @kevin-birdy 

image

This might be easiest to implement by adding a "Disabled" prop to the Horizontal child that's been rendered here. 

### Acceptance Criteria

* [ ] It should [SPEC_DETAIL]

### Other Resources

* Design mockup/prototype link
* Basecamp todo or link to "epic" on GitHub

### PR DESCRIPTION
**Pull request by burkeshartsis**
### What does this PR do, or why is it needed?
Highlights the active card in HorizontalContentFeeds so it is visually distinguishable from its siblings. 

### What design trade-offs/decisions were made?
None.

### How do I test this PR?
Tap into a content item with children in the app. Tap into a child scroll to the bottom to see it highlighted in the list with its siblings. 

### What automated tests did you write?
Saddly there is no test coverage over this component 😢 
## TODO:

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] Closes #1114 
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android
- [x] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by vinnyjth** [comment]
@burkeshartsis Looks great!!! I made one small tweak.

File: packages/apolloschurchapp/src/content-single/HorizontalContentFeed/index.js [hunk 1]
```diff
@@ -26,17 +26,21 @@ class HorizontalContentFeed extends Component {
...
-    >
-      <HorizontalContentCardConnected
-        contentId={get(item, 'id', '')}
-        disabled={get(item, 'id', '') === this.props.contentId}
-      />
-    </TouchableScale>
-  );
+  renderItem = ({ item }) => {
+    const itemId = get(item, 'id', '');
+    const disabled = get(item, 'id', '') === this.props.contentId;
```
Comment (burkeshartsis) [inline packages/apolloschurchapp/src/content-single/HorizontalContentFeed/index.js]: 👍

Comment (burkeshartsis) [inline packages/apolloschurchapp/src/content-single/HorizontalContentFeed/index.js]:
```suggestion
    const isDisabled = get(item, 'id', '') === this.props.contentId;
```
